hummingbot/connector/gateway/gateway_price_shim.py

```diff
@@ -38,17 +38,17 @@ class GatewayPriceDeltaEntry:
 
 class GatewayPriceShim:
     """
-    Developer / QA tool for modifying the apparent prices on DEX connectors (e.g. uniswap) during integration tests.
+    Developer / QA tool for modifying the apparent prices on DEX connectors (e.g. amm) during integration tests.
 
-    When the gateway price shim is enabled for a particular DEX trading pair (e.g. "WETH-DAI" on uniswap), the apparent
+    When the gateway price shim is enabled for a particular DEX trading pair (e.g. "WETH-DAI" on amm), the apparent
     price on the DEX will follow the prices on a different trading pair on another exchange (e.g. "ETH-USDT" on
     Binance). The price shim then exposes a function (`apply_price_delta()`) which allows the developer to modify the
     apparent prices on the DEX trading pair during live trading. This means the developer can manually control and
     inject arbitrage opportunities in an amm_arb trading session, for carrying out integration tests.
 
     How to use:
 
-    1. Set up an amm_arb strategy config that trades between a testnet trading pair (e.g. "WETH-DAI" on Uniswap Kovan),
+    1. Set up an amm_arb strategy config that trades between a testnet trading pair (e.g. "WETH-DAI" on Amm Kovan),
```

Comment:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed